docs/deep-dives/experimenter/rollouts.mdx

```diff
@@ -0,0 +1,85 @@
+---
+id: rollouts
+title: Rollouts
+slug: /rollouts
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Rollouts
+
+> 💡  Note: this feature is still in progress. See "Future work".
+
+Rollouts are single-branch experiments that differ from a traditional experiment in a number of ways.
```

Comment:
It's not really right to say a rollout is an experiment, maybe just say a rollout differs froman experiment in a number of ways.